config/lando.toml

```diff
@@ -0,0 +1,4 @@
+[auth]
+
+api_token="%SECRET%"
```

Comment:
So we have `settings.py` to set these ini prod for the ini files, but I don't think that's going to work here, is it?